src/pages/api/message/index.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable */
+import { retrieveUserMessage } from "~/lib/handlers/messages/MessageManager"
+import { configService } from '~/lib/handlers/config/Config';
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { createRouter } from 'next-connect';
+import { checkUser } from '~/lib/handlers/middleware/auth';
+
+const router = createRouter<NextApiRequest, NextApiResponse>();
+
+// Register route handlers to route
+// @ts-ignore
+router.get(checkUser, async (req: NextApiRequest, res: NextApiResponse) => {
+    // @ts-ignore
+    const user = req.user
+
+    if (!user) {
+        throw new Error('User is not logged in');
+    }
```

Comment:
I'm thinking if it'll be better to return the default message if not logged in. In case mattyu / our app can't retrieve or the user can't log in.